konrad/constants.py

```diff
@@ -85,7 +85,7 @@
         'units': '1',
         'standard_name': 'carbon_dioxide_mixing_ratio',
         'arts_name': 'abs_species-CO2',
-        'dims': ('plev',),
+        'dims': ('time', 'plev',),
```

Comment:
I know, it's a similar thing for eg the ozone profile, which is output at every timestep even if we choose to use a fixed ozone profile. I think it would be good to make this more flexible, so we only output things we need. Also, does CO2 need to output on all pressure levels, as we always consider it to be well mixed? Anyway, given that there are quite a few variables that we do want to output at each timestep and pressure level, maybe the couple that we're outputting unnecessarily are not making too much of an impact on the total data.

Reply:
I agree, that this should be handled more clearly. For the atmosphere this is a problem of the "convenient initialisation" from the constants. On could decrease the dimensions afterwords. But I guess it is fine to leave that for a general refactoring of the data handling

Reply:
Agreed